### PR TITLE
Feature/ik/text spotting3

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/text_spotting_evaluator.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 from functools import partial
 from collections import OrderedDict
 import numpy as np
+from scipy.special import softmax
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/text_spotting_evaluator.py:21]: could you please make scipy is optional requirement?
there is some trouble to install it on some target platforms where AC used for validation (i mean yocto now).
also as I know softmax was supported only in 1.2 release, it means that we need change lower bound version in requirements files.

Comment (Ilya-Krylov) [reply tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/text_spotting_evaluator.py:21]: "there is some trouble to install it on some target platforms where AC used for validation (i mean yocto now)" 
What should I do with this info? Should I continue use scipy or what?

Comment (Ilya-Krylov) [reply tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/text_spotting_evaluator.py:21]: is ok if I put it here https://github.com/openvinotoolkit/open_model_zoo/blob/develop/tools/accuracy_checker/requirements.in?

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/text_spotting_evaluator.py:21]: First of all, I need to understand, is it possible to get the same result without scipy usage?
Decision to continue use scipy or not is up to you. 
but if you want to stay it, please look how we make deal with sklearn in this case
https://github.com/openvinotoolkit/open_model_zoo/blob/develop/tools/accuracy_checker/accuracy_checker/metrics/classification.py
It needs to not break validation other models on that system where probably your model currently does not work at all.

Comment (Ilya-Krylov) [reply tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/text_spotting_evaluator.py:21]: I can implement softmax, but as far as I can see scipy is https://github.com/openvinotoolkit/open_model_zoo/blob/develop/tools/accuracy_checker/requirements.in already.

Comment (Ilya-Krylov) [reply tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/text_spotting_evaluator.py]: removed scipy

**Comment by eaidova** [comment]
@Ilya-Krylov please remove soft link on old AC config:
```
6:51:20  + /home/jenkins/workspace/omz/precommit/accuracy_checker_ubuntu18/venv/bin/yamllint configs/text-spotting-0002.yml
16:51:20  [Errno 2] No such file or directory: 'configs/text-spotting-0002.yml'
```

File: tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn_with_text.py [hunk 2]
```diff
@@ -33,10 +33,6 @@ def parameters(cls):
                 description="Name of output layer with information about classes.",
                 optional=False
             ),
-            'scores_out': StringField(
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn_with_text.py:36]: please keep code for support 0002. We still have it in quantization scope...

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn_with_text.py:36]: just make scores_out is optional

**Comment by Ilya-Krylov** [comment]
> @Ilya-Krylov please remove soft link on old AC config:
> 
> ```
> 6:51:20  + /home/jenkins/workspace/omz/precommit/accuracy_checker_ubuntu18/venv/bin/yamllint configs/text-spotting-0002.yml
> 16:51:20  [Errno 2] No such file or directory: 'configs/text-spotting-0002.yml'
> ```

So remove or return back?

**Comment by Ilya-Krylov** [comment]
@eaidova @vladimir-dudnik  I returned back support of text-spotting-0002 in accuracy checker, documentation, demo. Looks like I need to create new PR as soon as text-spotting-0002 is excluded from open model zoo. Am I right?

File: tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn_with_text.py (hunk 2 above)
Comment (Ilya-Krylov) [reply tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn_with_text.py:36]: fixed

File: models/intel/text-spotting-0003/description/text-spotting-0003.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+The text-spotting-0003-detector model is a Mask-RCNN-based text detector with ResNet50 backbone and additional text features output.
+
+| Metric                                        | Value     |
+|-----------------------------------------------|-----------|
+| GFlops                                        | TBD       |
+| MParams                                       | TBD       |
+
+
+### Performance
```
Comment (vladimir-dudnik) [inline models/intel/text-spotting-0003/description/text-spotting-0003.md:36]: we are removing empty sections from model documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please remove empty section here as well

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]

File: models/intel/text-spotting-0003/description/text-spotting-0003.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+*Hmean Word spotting* is defined and measured according to the
+[Incidental Scene Text (ICDAR2015) challenge](https://rrc.cvc.uab.es/?ch=4&com=introduction).
+
+## Detector model specification
+
+The text-spotting-0003-detector model is a Mask-RCNN-based text detector with ResNet50 backbone and additional text features output.
+
+| Metric                                        | Value     |
+|-----------------------------------------------|-----------|
+| GFlops                                        | TBD       |
```
Comment (vladimir-dudnik) [inline models/intel/text-spotting-0003/description/text-spotting-0003.md:32]: are these metric values known at this stage?

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]

**Review by vladimir-dudnik — APPROVED** [review]

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]
do not merge until accuracy validation release criteria met

**Comment by vladimir-dudnik** [comment]
release criteria achieved with the latest openvino build

**Review by vladimir-dudnik — APPROVED** [review]

**Comment by Ilya-Krylov** [comment]
@vladimir-dudnik I fixed documentation a bit please have a look. Is your approval is enough for merge?

**Review by eaidova — APPROVED** [review]
